action.yml

```diff
@@ -48,8 +48,8 @@ inputs:
     description: "The native-debug-symbols.zip file or folder that contains your debug symbols"
     required: false
   changesNotSentForReview:
-    description: "Indicates that the changes in this edit will not be reviewed until they are explicitly sent for review from the Google Play Console"
-    default: 'false'
+    description: "Indicates that the changes in this edit will not be reviewed until they are explicitly sent for review from the Google Play Console. Set to 'auto' to automatically retry with this flag if Google Play requires it. Defaults to 'false'"
```

Comment:
I don't think we need to include "Defaults to x", since we have the default explicitly captured below